app/i18n/en/admin.php

```diff
@@ -16,7 +16,7 @@
 		'allow_anonymous_refresh' => 'Allow anonymous refresh of the articles',
 		'api_enabled' => 'Allow <abbr>API</abbr> access <small>(required for mobile apps and sharing user queries)</small>',
 		'form' => 'Web form (traditional, requires JavaScript)',
-		'http' => 'HTTP (for advanced users with HTTPS)',
+		'http' => 'HTTP (advanced: managed by Web server, OIDC, SSO…)',
```

Comment:
The AP stylebook and a few others no longer recommend capitalizing web fwiw. (I prefer lowercase, but that might just be my non-English bias. They like capitalizing things a lot more than we do.)

Reply:
I did not know. I am still old school on that, apparently :-) For me *Web* is more precise than *web* (as in spider web):
* https://en.wiktionary.org/wiki/web
* https://en.wiktionary.org/wiki/Web

And this is what we have everywhere else in the current code base.
Do you use an upper case in *the Web* but not in *Web server*? Or in none of them?

Reply:
None. Which most style guides have also switched to about a decade ago. But if the FreshRSS style guide says Web let it be Web. :-)

Reply:
> if the FreshRSS style guide says

Haha, I do think we have such a thing yet, but let's come back to it another time if it bothers anyone